Beam/app/Http/Controllers/SearchController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Article;
+use App\Author;
+use App\Http\Requests\SearchRequest;
+use App\Http\Resources\ArticleSearchResource;
+use App\Http\Resources\SearchResource;
+use App\Segment;
+use Symfony\Component\Stopwatch\Stopwatch;
+
+class SearchController extends Controller
+{
+    public function search(SearchRequest $request)
+    {
+        $searchTerm = $request->query('term');
+
+        $stopwatch = new Stopwatch();
+        $stopwatch->start('search');
+        $searchResult['articles'] = Article::search($searchTerm)->with(['tags', 'sections'])->get();
+        $time = $stopwatch->stop('search');
+        dd($searchResult['articles'], $time->getDuration());
```

Comment:
This one looks forgotten..

Reply:
its just for testing purposes, we got push mirroring on our repos between gitlab and github so once we create PR it also reflects all commits here... i have purposely stated WIP keyword in the latest commit because of that nature of commit 😄

Reply:
oh, ok :). sorry :)